### ISSUE TITLE
Link urls not translating

### ISSUE THREAD

**Issue by SuddenDev** [pruvious/pruvious#3]
So first of all: Wow, excellent work! I Was looking for something like this for the longest time and I'm a big fan! Whenever I start pushing it through it's paces, I'll give more feedback and will recommend it to people!

But now to one issue I've encountered is that the Link Field, seems to not pickup the different languages.

**Steps to reproduce:**
1. I added the translation settings from the docs [here](https://pruvious.com/docs/translations#configuration) 
2. I added the Default Button Example Component [here](https://pruvious.com/docs/blocks#example)
3. I added 2 pages (Home and About) and added the Button Block.
4. The translations and the lang switch work great, but the urls always seem to point to the default language. So in German in routes to `/about` instead of `/de/ueber-uns`, even though the route definitely exists.

But maybe I'm not doing everything right or I didn't see a config for it in the docs.

```
  "devDependencies": {
    "nuxt": "^3.10.0",
    "vue": "^3.4.15",
    "vue-router": "^4.2.5"
  },
  "dependencies": {
    "pruvious": "^3.10.0"
  }
```

Any ideas? Thanks in advance!

**Comment by murisceman** [comment]
Thank you so much for your kind words and support, it means a lot.

Regarding the issue, there was a bug where nested fields were only displaying suggestions from the primary language. I have just released a new version [v3.10.1](https://github.com/pruvious/pruvious/releases/tag/v3.10.1) with a hotfix, so you can try updating the ﻿`pruvious` package in your project. Additionally, you'll need to update the saved value in the link field of your button block. I believe the values in both languages are currently set to the same page ID (e.g., ﻿`pages:1`).

I haven't had the chance to test Pruvious on a real project involving multiple languages yet. If you encounter more bugs, I'll be happy to squash them!

**Comment by SuddenDev** [comment]
@murisceman Wow thank you for the super quick fix! I just tested it and now it works as imagined!
I'll keep you updated and if I find more bugs I'll let you know!
